test/test_registry.cxx

```diff
@@ -26,16 +25,19 @@ class RegistryTest : public ::testing::Test {
     return HomeDirectory;
   }
 
-  DataPipelineImpl_ *init_pipeline(bool use_local = true) {
+  DataPipeline *init_pipeline(bool use_local = true) {
 
     const ghc::filesystem::path config_path_ =
         ghc::filesystem::path(TESTDIR) / "config.yaml";
     const ghc::filesystem::path script_path_ =
         ghc::filesystem::path(TESTDIR) / "test_script.sh";
     APILogger->set_level(spdlog::level::debug);
 
-    return new DataPipelineImpl_(config_path_, script_path_, token, spdlog::level::debug,
-                                 static_cast<RESTAPI>(use_local));
+    return new DataPipeline(
+        config_path_,
+        script_path_,
```

Comment:
Explicitly cast to string using: script_path_.string() to allow compiling on Windows.